packages/features/bookings/lib/handleNewBooking/test/workflow-notifications.test.ts

```diff
@@ -391,8 +391,7 @@ describe("handleNewBooking", () => {
         });
 
         expectWorkflowToBeTriggered({
-          // emailsToReceive: [organizer.email].concat(otherTeamMembers.map(member => member.email)),
-          emailsToReceive: [organizer.email],
+          emailsToReceive: [organizer.email].concat(otherTeamMembers.map((member) => member.email)),
```

Comment:
> can you also add a unit test for this? and make sure to test if all the hosts receive email

Instead of writing a new test uncommented this as this tests for receiving emails for all hosts.